python/ray/rllib/policy/dynamic_tf_policy.py

```diff
@@ -328,7 +329,12 @@ def fake_array(tensor):
             loss_inputs = [
                 (SampleBatch.CUR_OBS, self._obs_input),
             ]
-
+        # Only APPO agent has use_kl_loss in config. A bit hacky
+        if "use_kl_loss" in self.config:
```

Comment:
This shouldn't be here I think - I think this goes into `extra_compute_action_fetches`? 

Well, it should definitely not completely overwrite the previous `batch_tensors`. It should probably be part of the `postprocessed_batch` (or `dummy_batch`). 

cc @ericl

Reply:
After closer look, extra_compute_action_fetches and various methods like postprocess_trajectory add in an extra fetch from the workers, which will become input for the learner. "old_policy_behaviour_logits" is a special case where there is no extra fetch from the workers, but the learner needs the input. This "extra fetch" is generated in aso_learner.py, from old_worker. Are there work arounds for this?